### DISCUSSION TITLE
Can record, cant't control Hottop kn-8828b-2k+

### DISCUSSION THREAD

**Discussion by biangato** [artisan-roaster-scope/artisan#822] [Q&A] [answered]
Hi everyone,
thank you for your support!
On sunday i bought a Hottop KN-8828B-2K+. 
Even if SU (right click on timer) and control is activated, I cannot control the roaster via Artisan.
Recording profiles works. It's like it can read values but not write commands.
Using Artisan 2.4.0 and 2.4.6 on Win7 and Win10 (different Laptops) was not changing anything.
I tried a (old) step for guide from the artisan homepage for the COM settings, but it is not comparable, with artisan 2.4
Installing the driver recommended in the guide was not helping.

How can I figure out if the problem is on the roaster side or on Artisan side? How can I send commands without Artisan -> I haven't found helpful documents about that yet :-(

Thank you for your help!
Best regards
ToBi

**Comment by MAKOMO** [comment]
Please report only issue on the latest release which is v2.4.6. Older versions are not supported.
What about attaching here. Save it via menu Help >> Save Settings into a file artisan-settings.aset, rename it into artisan-settings.txt and drag-and-drop it here on a Github comment field for upload.

**Reply by biangato** [reply]
Hello Marko,
thank you for your reply. i am sorry for the delay.
[artisan-LinuxMint20.txt](https://github.com/artisan-roaster-scope/artisan/files/8146617/artisan-LinuxMint20.txt)


attached you'll find the settings for Linux Mint 20.3
The ones for Win 7 and Win 10 will follow in a few minutes.
THANK YOU FOR YOUR SUPPORT!
BR Tobias

**Reply by biangato** [reply]
[artisan-win7.txt](https://github.com/artisan-roaster-scope/artisan/files/8146626/artisan-win7.txt)

**Reply by biangato** [reply]
[artisan-win10.txt](https://github.com/artisan-roaster-scope/artisan/files/8146645/artisan-win10.txt)

**Reply by biangato** [reply]
in linux i followed the instruction from your instruction site to add the user to group. 
in Win7 was installing the driver recommended in a blog article for artisan 0.9.3. 
Win 10 is just as stock as possible

**Comment by MAKOMO** [comment, marked as answer]
I turned on my Hottop 2k+, loaded your win10 setup into Artisan v2.4.6 running on macOS, selected the correct serial port under Config >> Ports pointing to my connected Hottop, clicked ON and immediately got the ET/BT readings displayed in the corresponding LCDs, clicked the CONTROL-clicked (right-click) the timer LCD and got the message "super on" and then clicked CONTROL and the drum started to turn and all the control buttons like MOTOR OFF/ON and such worked as expected indicating full control of the machine. If control does not work for you, but data communication does as indicated by the ET/BT data received, there must be something wrong with your machine. Maybe Hottop changed the design, protocol or anything else, or your machine has a defect.

**Reply by biangato** [reply]
Dear Marko,

Thank you for supporting me!
...Maybe both. The roaster is about 5 years old - but something in my tummy expected a HW failure due to the symptoms. Thank you for checking!
Sadly Hottop not my answering mails. If I get more information, I will place it here.

Have a nice day
Best regards
Tobias
